project01/artur/application.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <cinder/app/App.h>
+#include <cinder/app/RendererGl.h>
+#include "snake.h"
```

Comment:
The order of include is wrong: always from most to least specific, i.e. first your includes, then libs, then STL